src/app/components/client/toolbar/AppPicker.tsx

```diff
@@ -215,109 +213,59 @@ export const AppPicker = () => {
   );
 };
 
-type AppPickerTriggerProps<T> = AriaMenuProps<T> &
-  MenuTriggerProps & {
-    label: string;
-    referringHost: string;
-  };
-const AppPickerTrigger = <T extends object>({
-  label,
-  referringHost,
-  ...otherProps
-}: AppPickerTriggerProps<T>) => {
-  const l10n = useL10n();
-  const appPickerTriggerState = useMenuTriggerState({
-    ...otherProps,
-    // TODO: Add unit test when changing this code:
-    /* c8 ignore start */
-    onOpenChange: (isOpen) => {
-      otherProps.onOpenChange?.(isOpen);
-      gaEvent({
-        category: "bento",
-        action: appPickerTriggerState.isOpen ? "bento-opened" : "bento-closed",
-        label: referringHost,
-      });
-    },
-    /* c8 ignore stop */
-  });
-
-  const triggerButtonRef = useRef<HTMLButtonElement>(null);
-  const { menuTriggerProps, menuProps } = useMenuTrigger<T>(
-    {},
-    appPickerTriggerState,
-    triggerButtonRef
-  );
+type AppPickerTriggerProps = {
+  label: string;
+  referringHost: string;
+} & AriaMenuProps<object> &
+  MenuTriggerProps &
+  AriaMenuTriggerProps;
 
-  const triggerButtonProps = useButton(
-    menuTriggerProps,
-    triggerButtonRef
-  ).buttonProps;
+// TODO: Add unit test when changing this code:
+/* c8 ignore start */
```

Comment:
This might be a good moment to add a unit test? Nothing too extensive - could be added to the dashboard test suite, adding a test that looks for a button with the name "⁨Mozilla⁩ apps and services", clicks it, then checking that an element with role `menu` exists. (And also that it did not exist before the button was clicked.)

(In any case, I wouldn't expect this ignore comment to wrap the entire component, as it already gets rendered in the dashboard, so the generic rendering code should already get executed.)